handbook/engineering/web/goals.md

```diff
@@ -1,47 +1,119 @@
+   - **Outcome:** A prioritized feature list built upon user research that can form the initial roadmap of the IDE extension. (This may reorder the later goals and add feature-based milestones.) 
 
-**_The web team has four areas of focus:_**
+1. Determine and build tracking for core performance and adoption metrics of the IDE extension. 
+   - **Outcome:** We can measure usage and adoption of the IDE extension separate it by existing/new users. 
 
-  1. **_Creating and maintaining a highly usable and intentionally designed webapp interface_**
-  1. **_Delivering the full, unique value of Sourcegraph [extensions](https://docs.sourcegraph.com/extensions)_**
-  1. **_Maintaining and expanding_ [_code host integrations_](https://docs.sourcegraph.com/dev/background-information/web/code_host_integrations)**
-  1. **_Developing code insights into an entirely new featureset_**
+1. The Sourcegraph IDE extension is a useable prototype and provides enough immediate value for developers to want to enable it.  
+   - **Outcome:** We see [*N<sub>8</sub>*][N8]% of existing Sourcegraph users enable and use the IDE extension. 
+   - **Outcome:** We get qualitative feedback from early adopters around what features are most useful or need work. 
 
-**Outcome**: 
+1. The Sourcegraph IDE extension is generally available and promoted across our pages and IDE extension registries. 
+   - **Outcome:** The Sourcegraph IDE extension increases DAU of Sourcegraph [*N<sub>9</sub>*][N9]% versus users who don't use the IDE extension. 
+   - **Outcome:** [*N<sub>10</sub>*][N10]% of new and existing Sourcegraph users use the IDE extension. 
 
-* All of these areas of focus are well-maintained, well-designed, well-documented, well-tested, performant, easy to set up, and leave users impressed. 
-* The webapp helps users discover and employ the full power of Sourcegraph effectively, with a high quality and highly usable interface. 
-* Soucegraph extensions and the extensions API provide powerful capabilities to users and a great experience for extension developers. 
-* Sourcegraph code host native integrations and browser extensions support the most common code hosts and browsers. 
-* Code insights expose the value of Sourcegraph's knowledge of your codebase to users at all levels of an organization. 
+1. The Sourcegraph IDE extension drives new sales and adoption of Sourcegraph. 
+   - **Outcome:** >[*N<sub>11</sub>*][N11] number of new "free tier" monthly first-time users started by using Sourcegraph in their IDE. 
+   - **Outcome:**  Qualitatively determined, at least three customer deals come in from companies excited about the Sourcegraph IDE extension or whose first Sourcegraph touchpoint was the IDE extension. 
 
-### Medium term
+### _Frontend Platform_ - make the web app UI consistent, accessible, and scalable 
```

Comment:
Is it intentional this is called "Frontend platform" and not "Web platform" (which is used below a lot)?

Reply:
`Frontend Platform` is the official name we are going forward with. We should update all `Web platform` references to `Frontend platform`